django/contrib/auth/templatetags/auth.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.hashers import UNUSABLE_PASSWORD_PREFIX, identify_hasher
+from django.template import Library
+from django.utils.html import format_html, format_html_join
+from django.utils.translation import gettext
+
+register = Library()
+
+
+@register.simple_tag
+def render_password_as_hash(value):
```

Comment:
I wonder if it would make sense to add some dedicated tests for this new tag under `tests/auth_tests/test_templatetags.py `. I realize it’s already being exercised indirectly through related forms and tests, but having a focused unit test suite in one place could help ensure we’re covering its behavior more explicitly and make future maintenance easier.

Also, unclear whether we should also add docs for this new template tag? :thinking: It's a matter of whether we expect/want people to reuse this or not. What do you think?

Reply:
I don't expect people to use it, it feels very specific for the `ReadOnlyPasswordHashWidget` which is also not documented

Reply:
Sounds good, thanks!